### PR TITLE
shell: inline the forward_command conditional to make it more explicit 

### PR DESCRIPTION
**Pull request by jelly**
This reverts commit 1bc7ee5d2054a320fcd0d2606ff31dc6f3750d9d.

### CODE REVIEW

**Review by martinpitt — APPROVED** [review]
Thanks! Looks good, but please squash-merge with a rationale. ("This was a no-op change, spotted by CodeQL" or so)

File: pkg/shell/base_index.js [hunk 1]
```diff
@@ -479,7 +476,7 @@ function Router(index) {
                 return;
 
             /* Only control messages with a channel are forwardable */
-            } else if (control.channel === undefined && !forward_command) {
+            } else if (control.channel === undefined) {
```
Comment (cockpituous) [inline pkg/shell/base_index.js:479]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18977-20230620-132626-075a58fe-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline pkg/shell/base_index.js:479]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18977-20230620-145037-075a58fe-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by martinpitt** [comment]
[TestMetricsPackages.testBasic](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18977-20230620-145037-075a58fe-fedora-38-devel/log.html#67-2) fails everywhere, that's suspicious.

**Comment by jelly** [comment]
> [TestMetricsPackages.testBasic](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18977-20230620-145037-075a58fe-fedora-38-devel/log.html#67-2) fails everywhere, that's suspicious.

Yeah, and I already retried it once. So hmm this requires investigation.

**Comment by jelly** [comment]
```
testlib.Error: timeout
wait_js_cond(ph_is_present("button:contains('Reconnect'):not([disabled]):not([aria-disabled=true])")): Uncaught (in promise) Error: condition did not become true
cdp: {'source': 'network', 'level': 'error', 'text': 'Failed to load resource: the server responded with a status of 401 (Authentication failed)', 'timestamp': 168733535332
0.6191, 'url': 'http://127.0.0.2:9191/cockpit/login', 'networkRequestId': '20964.12'}
log:
error: control channel  logout
```

Is this code actually compiled, different and actually used?!

**Comment by jelly** [comment]
@martinpitt I am also happy to move the `forward_command = True` assignment before the large `if / else if`.  No strong opinion here.

**Comment by martinpitt** [comment]
I thought about either leaving the body of the "if kill" empty with a comment, or moving the kill/close condition into the "return" branch. up to you. But this variable *is* dead, it's the dropping of the `if` that broke it.

**Review by martinpitt — APPROVED** [review]
That makes a lot more sense now, danjkewel! Triggering tests.

**Pull request merged**